x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/get_data_stream_details/index.ts

```diff
@@ -39,12 +39,14 @@ export async function getDataStreamSettings({
 
   const integration = dataStreamInfo?._meta?.package?.name;
   const lastBackingIndex = dataStreamInfo?.indices?.slice(-1)[0];
+  const indexTemplate = dataStreamInfo?.template;
 
   return {
     createdOn,
     integration,
     datasetUserPrivileges,
     lastBackingIndexName: lastBackingIndex?.index_name,
+    indexTemplate,
```

Comment:
Do we still need this in the response now that is not used by the update limit API?

Reply:
i still need it for sending users to Management Locator